dist/pact-consumer-js-dsl.js

```diff
@@ -1,6 +1,11 @@
 'use strict';
 
 var Pact = Pact || {};
+Pact.IsNodeJs = typeof module === 'object' && typeof module.exports === 'object';
+
+if (Pact.IsNodeJs) {
+  module.exports = Pact;
+}
 
```

Comment:
If I merge in this PR https://github.com/DiUS/pact-consumer-js-dsl/pull/27/files, I believe we won't need this.